### ISSUE TITLE
localtime is defined in the header, but has no implementation

### ISSUE THREAD

**Issue by zhuowei** [CraneStation/wasi-sysroot#20]
the `localtime` function is [present in wasi-sysroot's headers](https://github.com/CraneStation/wasi-sysroot/blob/e5f14be38362f1ab83302895a6e74b2ffd0e2302/libc-top-half/musl/include/time.h#L67), but not in the actual libc library, so programs calling localtime fails during linking:

```
#include <time.h>

int main() {
	time_t a;
	localtime(&a);
}
```

```
zhuowei@zhuowei-laptop:~/wasi-sdk/bin$ ./clang-8 localtime.c 
wasm-ld: error: /tmp/localtime-230b9c.o: undefined symbol: localtime
clang-8: error: lld command failed with exit code 1 (use -v to see invocation)
```

Either the localtime function should be removed from the header, or an implementation should be added to the libc.

**Comment by ericsink** [comment]
I bumped into this problem too.  Just to add more information:

The implementation does seem to be present:

libc-top-half/musl/src/time/localtime.c

So I'm guessing this is some kind of (possibly simple) problem with the build.

**Comment by kubkon** [comment]
As @ericsink rightly pointed out, the implementation is there, but the symbol is missing from the build as can be seen in [defined-symbols.txt](https://github.com/CraneStation/wasi-sysroot/blob/master/expected/wasm32-wasi/defined-symbols.txt) (or rather the lack of `localtime`).

**Comment by kubkon** [comment]
Fixed in #22.
